jmbitcoin/jmbitcoin/secp256k1_main.py

```diff
@@ -286,7 +414,10 @@ def privkey_to_pubkey_inner(priv, usehex):
     and return compressed/uncompressed public key as appropriate.'''
     compressed, priv = read_privkey(priv)
     #secp256k1 checks for validity of key value.
-    newpriv = secp256k1.PrivateKey(secret=priv)
+    if usehex:
+        newpriv = secp256k1.PrivateKey.from_hex(secret=bytes_to_native_str(priv))
+    else:
+        newpriv = secp256k1.PrivateKey(secret=bytes_to_native_str(priv))
```

Comment:
handling `usehex` here is not needed. It will be handled by the `hexbin` decorator and always be `False`.